lib/validators.js

```diff
@@ -144,6 +144,23 @@ validator.date = function(ctx, options, value) {
   }
 }
 
+validator.integer = function(ctx, options, value) {
+  if (typeof value === 'undefined' || value === null) return;
+
+  function isInteger(number) {
```

Comment:
I would like to suggest two modifications:
1. extract this function to outside the declaration of validator.integer;
1. at the moment the argument *number* is not used, modify the internal code to use number or change its name to *value* (which I prefer).